odin/validators.py

```diff
@@ -249,3 +250,47 @@ def validate_domain_part(self, domain_part):
                 pass
 
 validate_email_address = EmailValidator()
+
+
+class UUIDValidator(object):
```

Comment:
I'm not sure this validator is required it is essentially doing what fields do (eg decode the value). The field.to_python step occurs prior to validation so the only type that should be supplied to a validator is a UUID. The email/ip address fields are all strings that are required to be in a specific format.

Reply:
So all the validation should occur in `to_python()`?

Reply:
That's correct. Validators are called on values that have already been processed by `to_python`.